pkg/manager/manager.go

```diff
@@ -156,9 +157,9 @@ func (m *manager) ReleaseVF(conf *types.PluginConf, podifName, cid string, netns
 
 func getVfInfo(link netlink.Link, id int) *netlink.VfInfo {
 	attrs := link.Attrs()
-	for _, vf := range attrs.Vfs {
-		if vf.ID == id {
-			return &vf
+	for i := range attrs.Vfs {
```

Comment:
why was this change needed ? i think GC will not de-allocate the returned memory until its no longer referenced.
its because its not best practice to return a pointer to a var in for loop ? or was there actually a bug ?

Reply:
The range loop actually copies the objects on each iteration and we get a linter error: `rangeValCopy: each iteration copies 144 bytes`
I guess with the previous version of netlink, the size of the structure was less than 144 bytes and we were not hitting the linter threshold

Reply:
ack thx for clarifying